src/prefect/server/utilities/messaging/__init__.py

```diff
@@ -64,6 +65,43 @@ async def publish_data(self, data: bytes, attributes: Dict[str, str]):
         ...
 
 
+@dataclass
```

Comment:
I'm adding this implementation of a Publisher for use in tests, it's very similar to the in-memory one, but it doesn't actually publish anything to queues.